asm/macros/event.inc

```diff
@@ -2606,15 +2623,23 @@
 	@ 2 = Fast
 	@ 3 = Faster
 	.macro hidefollowernpc speed=1
+	checkfollowernpc
+	compare VAR_RESULT, TRUE
```

Comment:
Same here. If `FNPC_ENABLE_NPC_FOLLOWERS` is false I would just not even do `checkfollowernpc` but rather just have the macro nop